media-video/vdr/vdr-2.2.0-r4.ebuild

```diff
@@ -1,12 +1,12 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
 inherit eutils flag-o-matic toolchain-funcs
```

Comment:
do you really still need `eutils` in EAPI 7?

Reply:
tested it, indeed not necessary. removed.